### PR TITLE
Add thrift

### PR DESCRIPTION
**Pull request by ahawkins**
I'm not sure if I'm following the correct format. It seems there  is a `content.md` and `README.md` in the folder. They seem to share similar content. `README.md` usually contained the most information so I created
one for Thrift.

### CODE REVIEW

**Comment by ahawkins** [comment]
@tianon there you go! Thanks for the helpful guide on how to make this happen. Sorry it took so long :(

File: thrift/user-feedback.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+You are invited to contribute new features, fixes, or updates, large
+or small; I am always thrilled to receive pull requests, and do my
+best to process them as fast as I can.
+
+Before you start to code, I recommend discussing your plans through a
+[GitHub issue](https://github.com/ahawkins/docker-thrift/issues),
+especially for more ambitious contributions. This gives other
+contributors a chance to point you in the right direction, give you
+feedback on your design, and help you find out if someone else is
+working on the same thing.
```
Comment (tianon) [inline thrift/user-feedback.md]: This file seems like it's pretty close to the standard template -- is that right?

**Comment by tianon** [comment]
Generally LGTM, and don't worry -- it was the holidays after all.

Only the one comment from me; cc @yosifkit.

File: thrift/user-feedback.md (hunk 1 above)
Comment (yosifkit) [inline thrift/user-feedback.md]: Looks like just a `s/ we / I /g`?

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**